tarynmartin/personal-portfolio
# Feature/add content

#### What’s this PR do?
- Adds content to About section; written out and pdf of resume
- Style about section and some associated main styling
- Reorganize main to tab correctly
- Update fonts throughout site

#### Where should the reviewer start?
- Index.html for updated font links
- Main.js for new order and associated styling for changes to styling for main content section
- About.js and styling for changes

#### How should this be manually tested?
- in terminal, run npm test
- in browser, tab through buttons/links
- Click on about button to see new content
- Click on Resume button to see resume in a new tab
